modules/rostests/winetests/kernel32/sync.c

```diff
@@ -1180,7 +1180,8 @@ static HANDLE modify_handle(HANDLE handle, DWORD modify)
     return ULongToHandle(tmp);
```

Comment:
I understand doing this for tests that are actually flaky.
However i know that in kernel32:sync all that's flaky is the number of executed tests, whereas the number of failures is constant. Disabling (perfectly working) tests for that reason seems like a bad idea; I think the fix for this clearly belongs in Testman (either ignore test counts in general or supplement the blacklist functionality with a "gray-list" of tests that only vary in test count but not failures).

Reply:
I fixed this by adding ```disable_success_count``` to some tests that run a "random" number of times. This keeps the number of successes constant.